{{ cookiecutter.repo_name }}/.env

```diff
@@ -0,0 +1,46 @@
+# Environment variables go here, and can be read in by Python using the `python-dotenv`

```

Comment:
Add a comment about this file being tracked, similar to statement in .envrc

Reply:
Done